server_admin/topics/authentication/webauthn.adoc

```diff
@@ -34,107 +30,67 @@ An administrator carries out the following operations on the `Admin Console` :
 - Select `Webauthn Register` as `Required Action`.
 - Mark `Enabled` and `Default Action` checkbox.
 
-Enable 2FA by Webauthn Authenticator::
+===== Adding WebAuthn Authentication to a Browser Flow
 
-An administrator carries out the following operations on the `Admin Console` :
+* Select a realm, click on Authentication link, select the "Browser" flow
+* Make a copy of the built-in "Browser" flow. You may want to give the new flow a distinctive name, i.e. "WebAuthn Browser"
+* Using the drop down, select the copied flow
+* Delete the `WebAuthn Browser Browser - Conditional OTP` sub-flow using its `Actions` menu
 
-- Open the `Authentication -> Flows` tab.
-- Copy and create the new browser flow.
-- Add the `WebAuthn Authenticator` execution below the `User Password Form` execution.
-- Set `REQUIRED` to the `WebAuthn Authenticator` execution's requirement.
-- Open the `Authentication -> Bindings` tab.
-- Select the created new browser flow as `Browser Flow`.
-- Click `Save`.
+If you want to have WebAuthn required for all users:
 
-==== Register WebAuthn Authenticator
+* Using the `Actions` menu of the `WebAuthn Browser Forms`, click on `Add execution`
+* Select `WebAuthn Authenticator` using the drop down and click on "Save"
+* Set its Requirement to _Required_.
 
-The appropriate method to register a WebAuthen authenticator depends on if the user has or has not already registered an account on {project_name}.
+image:images/webauthn-browser-flow-required.png[]
 
-User without their account::
+Note that in this scenario, if a user doesn't have a WebAuthn credential, a required action will be set that forces him
+to register one.
 
-A user carries out the following operations :
+Alternatively, you can have users login with WebAuthn only if they have a WebAuthn credential registered, so instead of adding
```

Comment:
Use "log in" when it is verb.    Use "login" when it is a noun or adjective, such as "your login account."